src/scripts/import-open-api.ts

```diff
@@ -468,24 +470,73 @@ export interface ${pascal(name)}Response ${type}`;
   );
 };
 
+/**
+ * Validate the spec with ibm-openapi-validator (with a custom pretty logger).
+ *
+ * @param schema openAPI spec
+ */
+const validate = async (schema: OpenAPIObject) => {
+  // tslint:disable:no-console
+  const log = console.log;
```

Comment:
Maybe export this from [that other place you do it](https://github.com/contiamo/restful-react/pull/126/files#diff-2914e40119ca01b2d64663f0a82bb278R10)?

Reply:
It's more have only one `no-console`, I prefer to not export this hack ^^ So you need to ask yourself "is it a real usecase or just a console.log for debugging?"